rasa/core/channels/rocketchat.py

```diff
@@ -108,13 +108,14 @@ def from_credentials(cls, credentials: Optional[Dict[Text, Any]]) -> InputChanne
         if not credentials:
             cls.raise_missing_credentials_exception()
 
-        # pytype: disable=attribute-error
+        # rasa-16-to-rasa-17
         return cls(
             credentials.get("user"),
             credentials.get("password"),
             credentials.get("server_url"),
         )
-        # pytype: enable=attribute-error
+
+    # rasa-16-to-rasa-17
```

Comment:
Does it work like a `begin ignore` / `end ignore` or can we just drop the 2nd `#rasa-16-to-rasa-17`?

Reply:
I have no idea what this is 🤷

Reply:
Ok, then let's remove the new version and old version of the comments?

Reply:
soooo, apparently I introduced it myself in https://github.com/RasaHQ/rasa/pull/6470/commits/7fa00f42d5621b4ae10a8d179c0892b0f031929c... I've reverted and fixed it